pontoon/teams/static/css/team.css

```diff
@@ -68,6 +68,19 @@
   display: none;
 }
 
+#request-projects-note {
+  margin-top: 10px;
+  display: none;
+}
+
+#request-projects-note p {
+  margin-bottom: 5px;
```

Comment:
We should reuse style from a similar use case in the UI (e.g. Zarro Bugs found in the Bugs tab). Which would be easier if we had a proper theme.

```css
#request-projects-note p {
  font-style: italic;
  color: #AAAAAA;
  text-align: center;
}
```